specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/mfe.json

```diff
@@ -0,0 +1,1719 @@
+      "description": "Details of an AssetUri.",
+      "type": "object",
+      "properties": {
+        "path": {
+          "description": "The path of file/directory.",
+          "type": "string"
+        },
+        "isDirectory": {
+          "description": "Whether the path defines a directory or a single file.",
+          "type": "boolean"
+        }
+      }
+    },
+    "ModelVersion": {
+      "description": "Details of an ModelDto",
+      "type": "object",
+      "properties": {
+        "stage": {
+          "description": "Model asset stage.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The Name of the Asset.",
+          "type": "string",
+          "readOnly": true
+        },
+        "version": {
+          "description": "The Asset version assigned by service.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The Asset description text.",
+          "type": "string"
+        },
+        "datastoreId": {
+          "description": "The Asset datastoreId",
+          "type": "string"
+        },
+        "tags": {
+          "description": "The Asset tag dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "The Asset property dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "assetPaths": {
+          "description": "Details of asset path\r\nAdding list of back compat",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetPath"
+          }
+        },
+        "linkedResourceIds": {
+          "description": "Associated ARM resources. Key is the link type, value is a list of ARM IDs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "ModelVersionResource": {
+      "description": "Azure Resource Manager resource Envelope",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ModelVersion"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "ModelContainer": {
+      "type": "object",
+      "properties": {
+        "latestVersions": {
+          "description": "Latest Versions for all assets",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelVersionResource"
+          }
+        },
+        "name": {
```

Comment:
"name" is a top level property on the resource.  If you want a "displayName" or some other name here then please use a different property name